autofs-5.1.3 - improve description of mount_nfs_default_protocol

From: Ian Kent <raven@themaw.net>

The description of the mount_nfs_default_protocol configuration
option is not clear, it sounds like it's a setting that causes
autofs to use the specified NFS version as a mount default which
it doesn't, so try and improve it.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG                      |    1 +
 man/autofs.conf.5.in           |   10 +++++++---
 redhat/autofs.conf.default.in  |   12 ++++++++----
 samples/autofs.conf.default.in |   12 ++++++++----
 4 files changed, 24 insertions(+), 11 deletions(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -264,6 +264,7 @@
 - be silent about nis domain not set.
 - make map source reference message debug only.
 - handle additional nfs versions in mount_nfs.c.
+- improve description of mount_nfs_default_protocol.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/man/autofs.conf.5.in
+++ autofs-5.0.7/man/autofs.conf.5.in
@@ -62,10 +62,14 @@ Maps are browsable by default (program d
 .TP
 .B mount_nfs_default_protocol
 .br
-Specify the default protocol used by
+Set the default protocol that
 .BR mount.nfs (8)
-(program default 3). Since we can't identify this default automatically
-we need to set it in the autofs configuration.
+uses when performing a mount (program default 3). Autofs needs to know
+the default NFS protocol that
+.BR mount.nfs(8)
+uses so it can do special case handling for its availability probe for
+different NFS protocols. Since we can't identify this default
+automatically we need to set it in the autofs configuration.
 .TP
 .B append_options
 .br
--- autofs-5.0.7.orig/redhat/autofs.conf.default.in
+++ autofs-5.0.7/redhat/autofs.conf.default.in
@@ -44,10 +44,14 @@ timeout = 300
 #
 browse_mode = no
 #
-# mount_nfs_default_protocol - specify the default protocol used by
-# 			       mount.nfs(8). Since we can't identify
-# 			       the default automatically we need to
-# 			       set it in our configuration.
+# mount_nfs_default_protocol - set the default protocol that mount.nfs(8)
+# 			       uses when performing a mount. Autofs needs
+# 			       to know the default NFS protocol that
+# 			       mount.nfs(8) uses so it can do special case
+# 			       handling for its availability probe for
+# 			       different NFS protocols. Since we can't
+# 			       identify the default automatically we need
+# 			       to set it in our configuration.
 #
 #mount_nfs_default_protocol = 3
 mount_nfs_default_protocol = 4
--- autofs-5.0.7.orig/samples/autofs.conf.default.in
+++ autofs-5.0.7/samples/autofs.conf.default.in
@@ -44,10 +44,14 @@ timeout = 300
 #
 browse_mode = no
 #
-# mount_nfs_default_protocol - specify the default protocol used by
-# 			       mount.nfs(8). Since we can't identify
-# 			       the default automatically we need to
-# 			       set it in our configuration.
+# mount_nfs_default_protocol - set the default protocol that mount.nfs(8)
+#			       uses when performing a mount. Autofs needs
+#			       to know the default NFS protocol that
+#			       mount.nfs(8) uses so it can do special case
+#			       handling for its availability probe for
+#			       different NFS protocols. Since we can't
+#			       identify the default automatically we need
+#			       to set it in our configuration.
 #
 #mount_nfs_default_protocol = 3
 #
